spacy/training/batchers.py

```diff
@@ -112,13 +113,12 @@ def minibatch_by_padded_size(
         The `len` function is used by default.
     """
     if isinstance(size, int):
-        size_ = constant_schedule(size)
+        size_ = itertools.repeat(size)  # type: Iterator[int]
```

Comment:
Can't this be a regular type hint attached to `size_`?

Reply:
Yep. Fixed.